File: flipt-java/src/main/java/io/flipt/api/flags/models/ListFlagsResponse.java
package io.flipt.api.flags.models;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.flipt.api.models.PaginatedResponse;
import java.util.List;

public class ListFlagsResponse extends PaginatedResponse {

  @JsonProperty("flags")
  private List<Flag> flags;

  public List<Flag> getFlags() {
    return flags;
  }

  public void setFlags(List<Flag> flags) {
    this.flags = flags;
  }
}

File: flipt-csharp/src/Models/ListFlagsResponse.cs
using System.Text.Json.Serialization;

namespace Flipt.Models;

/// <summary>
/// Represents the response from listing flags.
/// </summary>
public class ListFlagsResponse : PaginatedResponse
{
    [JsonPropertyName("flags")]
    public List<Flag>? Flags { get; set; }
}
